chatkit-core/src/main/kotlin/com/pusher/chatkit/cursors/CursorSubscriptionEvent.kt

```diff
@@ -1,12 +1,12 @@
 package com.pusher.chatkit.cursors
```

Comment:
CursorSubscriptionEvent and CursorSubscriptionEventParser also is related to the backend API, can we place it in api subpackage? Likewise for other subscription events/parser.

Reply:
Likewise for the few remaining Services.

Reply:
👍 fair point - have updated this. I found the `Android` package view has somewhat broken for the cursors and rooms package (as everything now is inside the api package) - but viewing with the `Project` mode is unaffected.